lib/spack/env/cc

```diff
@@ -271,75 +339,64 @@ for dep in "${deps[@]}"; do
     # Prepend include directories
```

Comment:
Yeah I'll change the comment. We have to append, because if we prepend we mess up any time the dependency has an included header with the same name as one of our headers.